src/server/views/dashboard/partials/table-header-cell.njk

```diff
@@ -0,0 +1,33 @@
+{% set nameToTitleMap = {
+  type: "Type",
+  country: "Country",
+  live: "Live",
+  actionNeeded: "Action",
+  lastAnnualReviewStartDate: "Last review",
+  nextAnnualReviewStartDate: "Next review",
+  admins: "Admins",
+  listId: "Settings"
+} %}
+
+{% set orderByToAriaAttributeMap = {
+  asc: "ascending",
+  desc: "descending",
+  'none': "none"
```

Comment:
unquoted none is treated like undefined or null for some reason, hence the inconsistent quoting